cli/cmd/cli_unix.go

```diff
@@ -32,7 +36,12 @@ var promptIconsFunc = func(icons *survey.IconSet) {
 // UpdateCommand returns the command that a user should run to update the cli
 // to the latest available version (unix specific command)
 func (c *cliState) UpdateCommand() string {
+	if os.Getenv("LW_HOMEBREW_INSTALL") != "" {
+		return `
+	 $ brew upgrade lacework-cli
+	`
```

Comment:
Same for this string, let us not use tabs or extra spaces:
```suggestion
  $ brew upgrade lacework-cli
`
```